src/commands/play.js

```diff
@@ -34,11 +34,11 @@ module.exports = {
         });
 
         try {
-            const { type, playlistName, tracks } = await music.load(util.isValidURL(query) ? query : `ytsearch:${query}`);
-            if (!tracks.length) return ctx.respond({
+               const result =  await music.load(util.isValidURL(query) ? query : `ytsearch:${query}`);
+            if (!result || !result.tracks.length)  return ctx.respond({
```

Comment:
```suggestion
            if (!result?.tracks.length)  return ctx.respond({
```
use optional chaining